kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableProcessingReport.java

```diff
@@ -23,6 +23,8 @@
 public class PurchasingAccountsPayableProcessingReport extends GeneralLedgerEntry {

```

Comment:
I have gone over all the correspondence of if this prop should be extended, and then foundation imports changed. I am of the mind that since this change is so simple and self-contained, I am in favor of leaving this as is. Having said that, I believe each of these scenarios should be assessed on a case-by-case basis to make make the best call. I am _not_ requesting a change here.